recipe/meta.yaml

```
@@ -1,41 +1,38 @@
-{% set version = "4.1.3" %}
+{% set version = "5.0.5" %}
 
 package:
   name: moto
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/m/moto/moto-{{ version }}.tar.gz
-    sha256: c8200ccaa9440c2e9daa0bd5e0bd768a719db5a2c82ea8d782f0e3fa09a3c5e2
+    sha256: 2eaca2df7758f6868df420bf0725cd0b93d98709606f1fb8b2343b5bdc822d91
 
 build:
   number: 0
   # there seem to be missing docker-py >=2.5.1 and cfn-lint >=0.4.0 for s390x
   skip: True  # [py<37 or s390x]
   entry_points:
     - moto_server = moto.server:main
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=40.6.0
     - wheel
   run:
     - python
-    # we need to bound this since 198 changed some of the api
     - boto3 >=1.9.201
-    - botocore >=1.12.201
+    - botocore >=1.14.0
     - cryptography >=3.3.1
     - requests >=2.5
     - xmltodict
     - werkzeug >=0.5,!=2.2.0,!=2.2.1
     - python-dateutil >=2.1,<3.0.0
-    - responses >=0.13.0
+    - responses >=0.15.0
     - jinja2 >=2.10.1
-    - python-jose >=3.1.0,<4.0.0
-    - ecdsa !=0.15
     - docker-py >=2.5.1
```

```suggestion
    - docker-py >=3.0.0
```
https://github.com/getmoto/moto/blob/5.0.5/setup.cfg#L50C13-L50C18

---

Should we keep `2.5.1` because it is the lowest viable version needed for proxy listed [here](https://github.com/getmoto/moto/blob/5.0.5/setup.cfg#L65) or will the older versions affect something else?